### PR TITLE
Updating Captcha Checker Function

### PR DESCRIPTION
**Pull request by TravisHunting**
Modified captcha function (simplified) and added a call for it under scrapeImageLinksFromIssue

### CODE REVIEW

File: comicScraper.py [hunk 1]
```diff
@@ -38,18 +39,12 @@
     'User-Agent': 'Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:52.0) Gecko/20100101 Firefox/52.0'
     }
 
-# TODO: double check this is the list/array that are you human shows up in
-def dealWithCaptcha(imageLinks):
-    # check for single url
-    if "/Special/AreYouHuman" in imageLinks:
+def checkForCaptcha(line,baseUrl):
```
Comment (AustinHunting) [inline comicScraper.py]: space between `line` and `baseUrl`

File: comicScraper.py [hunk 2]
```diff
@@ -109,6 +104,10 @@ def scrapeImageLinksFromIssue(url):
             imageUrl = extractImageUrlFromText(line)
             imageLinks.append(imageUrl)
 
+        if checkForCaptcha(line,url):
+            # Note: currently we are not actually solving the captcha so this will start an endless loop
+            return scrapeImageLinksFromIssue(url)
```
Comment (AustinHunting) [inline comicScraper.py]: we should add a loop here that kills the script if the function gets called more than a couple of times, the `start an endless loop` part is worrying to me.

File: comicScraper.py [hunk 3]
```diff
@@ -109,6 +104,10 @@ def scrapeImageLinksFromIssue(url):
             imageUrl = extractImageUrlFromText(line)
             imageLinks.append(imageUrl)
 
+        if checkForCaptcha(line,url):
```
Comment (AustinHunting) [inline comicScraper.py]: does this get called for every image link?

**Comment by AustinHunting** [comment]
few comments but mostly good

File: comicScraper.py (hunk 3 above)
Comment (TravisHunting) [reply comicScraper.py]: It gets called for every line in the soup from the issue link

**Review by AustinHunting — APPROVED** [review]
looks good to me

**Pull request merged**